packages/formation/js/sidenav.js

```diff
@@ -91,8 +91,13 @@ class SideBarMenu {
     this.menu.setAttribute('aria-hidden', 'false');
     document.getElementsByTagName('body')[0].style.overflow = 'hidden';
     this.addCloseMenuListener();
-    // main trigger, switch focus to close button
-    this.closeControl.focus();
+    // Switch focus to close button. In Safari the focus was not always getting
+    // set when the menu opened. This event loop hack fixes the issue and
+    // ensures that the menu's close button gets focus when the sidenav menu
+    // opens.
+    setTimeout(() => {
+      this.closeControl.focus();
+    }, 0);
```

Comment:
If this is a known issue, can you add a link to somewhere where it's documented?

Reply:
you mean a link to the github issue in a code comment?

Reply:
I'll link to https://github.com/department-of-veterans-affairs/va.gov-team/issues/18916 which has a video showing focus not going to the close button like it should.

Reply:
That works!